packages/clarity-js/src/core/history.ts

```diff
@@ -12,24 +12,30 @@ export function start(): void {
     url = getCurrentUrl();
     count = 0;
     bind(window, "popstate", compute);
-    
+
     // Add a proxy to history.pushState function
-    if (pushState === null) { pushState = history.pushState; }
-    history.pushState = function(): void {
-        if (check()) {
+    // Add a proxy to history.pushState function
+    if (pushState === null) { 
```

Comment:
>pushState === null [](http://example.com/codeflow?start=8&length=18)

What happens if we download Clarity script twice on the page? Would it cause a weird edge case where we override the definition and then override it again because another instance will reset pushState etc. variables?

Reply:
it wouldnt cause any issues - history.pushstate will not be overriden again